cmd/pk-put/kvcache.go

```diff
@@ -346,7 +346,7 @@ func fileInfoToFingerprint(fi os.FileInfo) statFingerprint {
 		if clean := cleanSysStat; clean != nil {
 			// TODO: don't clean bad fields, but provide a
 			// portable way to extract all good fields.
-			// This is a Linux+Mac-specific hack for now.
+			// This is a Linux+Mac+Windows-specific hack for now.
```

Comment:
At which point does it become faster to list the platforms for which the hack does not exist? ;)

Reply:
sure.  Do we have an official list of platforms that Perkeep supports?  Because we could also just work on adding support for those platforms.

Reply:
Linux, Mac, Windows, FreeBSD probably.

Reply:
apparently, this seems to work fine on FreeBSD already (tested on FreeBSD 12.1), so I've updated this commit to just remove this "hack" comment entirely.  I've left the TODO in place however, since it does still seem like a good idea to extract good fields rather than clean bad fields.